### PR TITLE
Add searchability to filings/operations-log endpoint

### PR DESCRIPTION
**Pull request by hcaofec**
## Summary (required)

- Resolves [#3467](https://github.com/fecgov/openFEC/issues/3467)

- More data fields are added to the endpoint to expand its searchability _

## How to test the changes locally

Test this link and compare the result to sql query result
http://127.0.0.1:5000/v1/operations-log/?sort_hide_null=false&min_receipt_date=05%2F01%2F2018&sort_null_only=false&sort_nulls_last=false&sort=-report_year&per_page=20&page=1

### CODE REVIEW

File: tests/test_operation_logs.py [hunk 1]
```diff
@@ -34,4 +35,18 @@ def test_verified_reports(self):
         results = self._results(api.url_for(OperationsLogView, candidate_committee_id='02', status_num=1))
         self.assertEqual(len(results), 1)
         for result in results:
-            self.assertTrue(1, result['status_num'])
\ No newline at end of file
+            self.assertTrue(1, result['status_num'])
+
+    def test_receipt_date(self):
```
Comment (pkfec) [inline tests/test_operation_logs.py]: Well done @hcaofec! Could you add another test case to test the data range on receipt_date column? A simple test to make sure that the date range is picked up correctly.Here is an example to test the data range filter:
https://github.com/fecgov/openFEC/blob/develop/tests/test_filings.py#L35

Comment (hcaofec) [reply tests/test_operation_logs.py]: test case has been added.
@fec-jli @pkfec

**Comment by fec-jli** [comment]
good work.
Can you add test case for 'overage_end_date' and 'transaction_data_complete_date'?
same as test case for 'receipt_date'. Thanks

File: tests/test_operation_logs.py (hunk 1 above)
Comment (pkfec) [reply tests/test_operation_logs.py]: @hcaofec Looks good. I will merge this PR.

**Pull request merged**